README.md

```diff
@@ -15,6 +15,31 @@
 pest is a [PEG](https://en.wikipedia.org/wiki/Parsing_expression_grammar) parser
 generator with *simplicity* and *speed* in mind.
 
+## Usage
+You will need Cargo and Rust, follow the [installation instructions](https://www.rust-lang.org/en-US/downloads.html) 
+to get them.
+
+Add the following to `Cargo.toml`:
+
+```toml
+pest = "0.4"
+```
+
+and in your Rust `lib.rs` or `main.rs`:
+
+```rust
+#[macro_use]
+extern crate pest;
+```
+
+### Build & test locally
```

Comment:
I meant here to only use the "You can run the culculator..." sentence instead of the whole section, because the following two sections are important and should be easilt visible.